src/index.ts

```diff
@@ -59,15 +60,16 @@ const usePactWait = (alias: AliasType) => {
         intercept: flattenIntercept,
         testCaseTitle: `${testCaseTitle}`,
         pactConfig,
-        blocklist: headersBlocklist
+        blocklist: headersBlocklist,
+        providerState: providerState
       })
     })
   }
 }
 
 const requestDataMap: AnyObject = {}
 
-const usePactGet = (alias: string) => {
+const usePactGet = (alias: string, pactConfig: PactConfigType, providerState: string = '') => {
```

Comment:
`pactConfig` is a required field, so this would be a breaking interface change for the user.

Also I don't think it should be included?